src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -961,6 +961,41 @@ static void _updateInvalidViewSize(const Scene* scene, Box& vBox, float& w, floa
     if (!validHeight) h *= vBox.h;
 }
 
+
+static void _loadFonts(Array<EmbeddedFont>& fonts)
+{
+    if (fonts.empty()) return;
+
+    TaskScheduler::async(false);
+    for (auto font = fonts.begin(); font < fonts.end(); ++font) {
+        if (!font->src || !font->name) return;
+
+        auto src = font->src;
+        if (strncmp(src, "data:", sizeof("data:") - 1)) continue;
+        src += sizeof("data:") - 1;
+
+        //handling: 1) data:font/ttf;base64, 2) data:application/font-ttf;base64,
+        src = strstr(src, "ttf;base64,");
+        if (!src) {
+            TVGLOG("SVG", "The embedded font named \"%s\" not loaded properly.", font->name);
+            tvg::free(font->src);
+            font->src = nullptr;
+            continue;
+        }
+        src += sizeof("ttf;base64,") - 1;
+
+        char *decoded = nullptr;
+        auto size = b64Decode(src, strlen(src), &decoded);
```

Comment:
Should we consider utf-8 decoding?

Reply:
But font data is binary - base64 allows embedding this data inside a text based file. 
utf8 encodes unicode text, so it won't work for binary font data. Do you agree?